storage-xray-udp/src/main/java/zipkin2/storage/xray_udp/UDPMessageEncoder.java

```diff
@@ -121,11 +201,56 @@
       writer.endObject();
     }
 
+    Integer errorStatus = httpResponseStatus;
```

Comment:
We have standard http tag for this. http.status_code I believe. For input tags, reuse zipkin ones and namespace (ex xxxx.foo) where exist

Reply:
If there is an http error and available `http.status_code`, I am using it. But there might be some other errors as well. For example a DB exception or something else. For that part our instrumentation codes must provide `error_status`.

Reply:
> For input tags, reuse zipkin ones and namespace (ex xxxx.foo) where exist
I could not get this, can you expand please

Reply:
Mainly it is getting cognitively difficult to follow things. Let's try not to move ahead of the ball, as there's an advantage in x-ray which is that it is model based. Instead of creating error_status which isn't defined anywhere in zipkin, for possible correlated use between http and sql, perhaps define them separately, in namespaces such as http.status_code (and whatever it is for sql, ex sql.status_code). Eventhough I have more time than others I'm having trouble following this, so lets try and keep it organized as possible and using zipkin conventions whereever, and without support for hypothetical things (ex only code things you have a test case and actual brave instrumentation to support)

Reply:
There might be other services can cause status_code. How can I add status_code for a redis call or a method instrumentation. If you believe that it is confusing. Let's 
change `error_status` to `xray.error_status` since it is a top level property of Subsegments.